vuepress/docs/next/tutorials/create/mfe/react.md

```diff
@@ -90,6 +92,7 @@ You’re now using a custom element to display a React app.
 
 1. From the root bundle directory, generate the Docker image:
    ``` sh
+   cd simple-bundle
```

Comment:
I don't think you need this command. If you're in the MFE subdirectory, you would either use cd ../.. - or simply leave the comment that they should work from the root bundle directory.